### PR TITLE
fix: DEV-1503: Fix loading all tasks from project

### PR DESCRIPTION
**Pull request by triklozoid**

### CODE REVIEW

File: label_studio_sdk/project.py [hunk 1]
```diff
@@ -520,16 +521,26 @@ def get_tasks(
...
+                    view_id=view_id,
+                    selected_ids=selected_ids,
+                    only_ids=only_ids,
+                    page=page,
+                    page_size=100
+                )
+                result += data['tasks']
+                page += 1
+            # we'll get 404 from API on empty page
+            except LabelStudioException:
```
Comment (niklub) [inline label_studio_sdk/project.py:541]: we need logging here, and a more broad exception handler should prevent infinite loop

File: label_studio_sdk/project.py [hunk 2]
```diff
@@ -618,8 +629,10 @@ def get_paginated_tasks(
         if only_ids:
             params['include'] = 'id'
 
-        response = self.make_request(
-            'GET', '/api/dm/tasks', params)
+        try:
+            response = self.make_request('GET', '/api/tasks', params)
```
Comment (niklub) [inline label_studio_sdk/project.py:633]: this should be released only after corresponding LS/LSE release?

**Review by niklub — CHANGES_REQUESTED** [review]

File: label_studio_sdk/project.py (hunk 1 above)
Comment (triklozoid) [reply label_studio_sdk/project.py:541]: > we need logging here

It's not an error, we expect, that after the last page we'll get 404, what type of logging it should be? 

> more broad exception handler should prevent infinite loop

I can't see ability for infinite loop there, for LabelStudioException we have a `break` statement and any other exception will throw up and interrupt the loop.

File: label_studio_sdk/project.py (hunk 2 above)
Comment (triklozoid) [reply label_studio_sdk/project.py:633]: Yes.

File: label_studio_sdk/project.py (hunk 1 above)
Comment (triklozoid) [reply label_studio_sdk/project.py]: Added logging.

**Review by makseq — APPROVED** [review]

**Review by makseq — APPROVED** [review]

**Pull request merged**